### PR TITLE
[paligemma2] snippet fixes

### PR DESCRIPTION
**Pull request by pcuenca**

### CODE REVIEW

File: paligemma2.md [hunk 1]
```diff
@@ -163,16 +163,19 @@ After that, you can run inference like this:
...
+from PIL import Image
+import requests
 
 model_id = "google/paligemma2-10b-ft-docci-448"
 model = PaliGemmaForConditionalGeneration.from_pretrained(model_id)
 model = model.to("cuda")
 processor = AutoProcessor.from_pretrained(model_id)
 
-prompt = "caption en"
+prompt = "<image>caption en"
```
Comment (pcuenca) [inline paligemma2.md:174]: otherwise we get a warning (and it's good to show)

File: paligemma2.md [hunk 2]
```diff
@@ -163,16 +163,19 @@ After that, you can run inference like this:
...
 image_file = "https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/cats.png"
-raw_image = Image.open(requests.get(image_file, stream=True).raw)
-inputs = processor(prompt, raw_image, return_tensors="pt").to(“cuda”)
+raw_image = Image.open(requests.get(image_file, stream=True).raw).convert("RGB")
+
+inputs = processor(prompt, raw_image, return_tensors="pt").to("cuda")
```
Comment (pcuenca) [inline paligemma2.md:178]: pasting the code had decided to use weird quote chars, not `"` :)

**Review by evilmerve123 — APPROVED** [review]
Thank you!

**Review by merveenoyan — APPROVED** [review]
sorry for evil account interruption

**Comment by pcuenca** [comment]
Thanks for the quick review @evilmerve123 and @merveenoyan!

**Pull request merged**